src/ax/saga/workflow.go

```diff
@@ -14,75 +15,122 @@ type Workflow struct {
 	ErrorIfNotFound
 	CompletableByData
 
-	Prototype   Data
-	Triggers    ax.MessageTypeSet
-	NonTriggers ax.MessageTypeSet
-	Handle      typeswitch.Switch
+	Prototype     Data
+	Triggers      ax.MessageTypeSet
+	NonTriggers   ax.MessageTypeSet
+	CommandHandle typeswitch.Switch
```

Comment:
These names should probs be `HandleCommand` and `HandleEvent`, for maximum English overdrive.